lib/board.rb

```diff
@@ -0,0 +1,51 @@
+class Board
+	attr_accessor :cells
+
+	def initialize
+		@cells = [" "]*9
+	end
+
+	def reset!
+		@cells = [" "]*9
```

Comment:
I would just call `self.reset!` here and delegate to that method to create the default empty board.